cylc/flow/hostuserutil.py

```diff
@@ -113,7 +113,7 @@ def _get_host_info(self, target=None):
         """Return the extended info of the current host."""
         if target not in self._host_exs:
             if target is None:
-                target = socket.getfqdn()
+                target = socket.gethostname()
```

Comment:
Any particular reason for this change? (They seem to always return the same result on Linux?).  If `gethostname()` did return just the hostname, then the docstring is now wrong (if that's what "extended info" means...

Reply:
Dammit, that’s my mac patch again, shouldn’t have rushed this in.